hardware/raspberrypi/bootmodes/usb.md

```diff
@@ -7,6 +7,6 @@ There are two separate boot modes for USB (only available on certain models):
   * [USB mass storage boot](msd.md)
   * [Network boot](net.md)
 
-The choice between the two boot modes is made by the firmware at boot time when it reads the OTP bits. There are two bits to control USB boot: the first enables device boot and is enabled by default on all Raspberry Pi computers. The second bit enables USB host boot; if this bit is also set, then the processor reads the OTGID pin to decide whether to boot as a host (driven to zero as on the Raspberry Pi Model B) or as a device (left floating). The Pi Zero has access to this pin through the OTGID pin on the USB connector, and the Compute Module has access to this pin on the edge connector.
+The choice between the two boot modes is made by the firmware at boot time when it reads the OTP bits. There are two bits to control USB boot: the first enables USB device boot, and the second enables USB host boot. If the USB host boot mode bit is set, then the processor reads the OTGID pin to decide whether to boot as a host (driven to zero as on the Raspberry Pi Model B) or as a device (left floating). The Pi Zero has access to this pin through the OTGID pin on the USB connector, and the Compute Module has access to this pin on the edge connector.
```

Comment:
Given that the USB host boot mode has to be manually enabled on all Pis without a BCM2837B0, I think it might be worth leaving in the text clarifying that USB device boot is enabled by default (i.e. _doesn't_ need to be manually enabled).
AFAIK USB device boot **is** enabled on all Pis, it's just that on Pi's with a built-in USB hub (i.e. all the B/B+ models) the downstream hub forces the SoC's USB port into host mode, and prevents it from being able to switch into device mode.  But I'm no USB expert... :wink: 

There's also a little bit of duplication between the text here and the more-detailed info in the bootflow.md page, but that's probably not worth worrying about?

Reply:
I've added a bit back so it now reads 

> the first enabled USB device boot which is enabled by default

In other words, the net effect is to remove the bit that says "on all Raspberry Pi's". If folk want to desolver the LAN7500 / 9500 chip to get device boot working, the page still provides enough information to tell them that this can be done. It just no longer suggests that device mode works on all models of Pi (because it doesn't).